docs/install.rst

```diff
@@ -18,24 +21,38 @@ Installing BridgeStan is as simple as ensuring that the above pre-requisites are
 the source repository.
 
 If you have ``git`` installed, you may do this by navigating to the folder you'd like
-BridgeStan to be in and running ``git clone git@github.com:roualdes/bridgestan.git``.
-
-If you do not have ``git`` installed, you can download a `.zip` file containing the repository
-`here <https://github.com/roualdes/bridgestan/archive/refs/heads/main.zip>`__. Unzip this
-file into the folder you would like BridgeStan to be in.
+BridgeStan to be in and running ``$ git clone --recurse-submodules https://github.com/roualdes/bridgestan.git``.
 
 After this, BridgeStan is installed. You can test a basic compilation by opening
 a terminal in your BridgeStan folder and running
 
 .. code-block:: shell
 
     # MacOS and Linux
-    make CMDSTAN=/path/to/cmdstan/here/ test_models/multi/multi_model.so
+    make test_models/multi/multi_model.so
     # Windows
-    mingw32-make.exe CMDSTAN=C:/path/to/cmdstan/with/forward/slashes/ test_models/multi/multi_model.so
+    mingw32-make.exe test_models/multi/multi_model.so
 
 This will compile the file ``test_models/multi/multi.stan`` into a shared library object for use with BridgeStan.
+This requires internet access the first time it is run.
+
+
+Using custom Stan versions
+--------------------------
+
+If you wish to use BridgeStan for an older released version, all you need to do is
+
+1. Set ``STANC3_VERSION`` in ``make/local`` to your desired version, e.g. ``v2.26.0``
+2. Go into the ``stan`` submodule and run ``git checkout release/VERSION``, e.g. ``release/v2.26.0``
+3. Also in the ``stan`` submodule, run `make math-update`
+
+To return to the version of Stan currently used by BridgeStan, you can run ``make stan-update`` from the top level directory
```

Comment:
Does it matter if `make stan-update` is run before removing the `STANC3_VERSION` variable?  Based on the itemized list above, I would assume removing `STANC3_VERSION` should happen first, before running `make stan-update`, but that's not how I read this sentence.

Reply:
Good point, there are actually a garden of forking paths of orders one could do here. The important thing for stanc is actually running `make clean` or deleting `bin/stanc` to force a re-download, so that should be on the list!